src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1056,11 +1056,15 @@ private function addDefaultParametersMethod()
 
         $php = array();
         $dynamicPhp = array();
+        $normalizedParams = array();
 
         foreach ($this->container->getParameterBag()->all() as $key => $value) {
             if ($key !== $resolvedKey = $this->container->resolveEnvPlaceholders($key)) {
                 throw new InvalidArgumentException(sprintf('Parameter name cannot use env parameters: %s.', $resolvedKey));
             }
+            if ($key !== $lcKey = strtolower($key)) {
+                $normalizedParams[] = str_repeat(' ', 8).sprintf('%s => %s,', $this->export($lcKey), $this->export($key));
```

Comment:
not sure using str_repeat is better than putting spaces in the sprintf (same below)